pages/api/analytics/charts/totalSupply.ts

```diff
@@ -1,45 +1,87 @@
-import DuneClient, { toChartData, jobsLookup } from "../../../../lib/dune";
+import Redis from "ioredis";
```

Comment:
I think this is a different use case for a cache. In react-query it would prevent an individual client from remaking calls but this is making it so any client gets the same cache.